### PR TITLE
ADD CSJ RNN pretrained model 

### PR DESCRIPTION
**Pull request by jnishi**
I uploaded the CSJ RNN pre-trained model [here](https://drive.google.com/file/d/1ALvD4nHan9VDJlYJwNurVr7H7OV0j2X9/view?usp=sharing).

It is useful for CTC Segmentation, though it is worse than the transformer model in terms of accuracy.

### CODE REVIEW

**Comment by sw005320** [comment]
Many thanks!
